uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document.spec.js

```diff
@@ -393,11 +393,32 @@ define([
           describe('user is admin', function () {
             beforeEach(function () {
               controller.role = 'admin';
-              controller.confirm();
             });
 
-            it('sets document status to Approved', function () {
-              expect(DocumentService.save).toHaveBeenCalledWith(jasmine.objectContaining({ status_id: '3' }));
+            describe('when uploaded files sizes are smaller than max file limit', function () {
+              beforeEach(function () {
+                controller.confirm();
+              });
+
+              it('sets document status to Approved', function () {
+                expect(DocumentService.save).toHaveBeenCalledWith(jasmine.objectContaining({ status_id: '3' }));
+              });
+
+              it('checks files as valid and returns true', function () {
```

Comment:
This is not needed since this is a duplication of the test above.

Reply:
Right! removed